substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1775,12 +1756,12 @@ impl<T: Config> historical::SessionManager<T::AccountId, Exposure<T::AccountId,
 	}
 }
 
-impl<T: Config> historical::SessionManager<T::AccountId, ()> for Pallet<T> {
-	fn new_session(new_index: SessionIndex) -> Option<Vec<(T::AccountId, ())>> {
+impl<T: Config> historical::SessionManager<T::AccountId, Existence> for Pallet<T> {
```

Comment:
Similarly, this code will be removed once we revert https://github.com/paritytech/polkadot-sdk/pull/7937

Reply:
I can get rid of them here as well. They are used by the test runtimes.

Reply:
I’d prefer to keep it. New solo chains that never relied on exposures can simply use Existence and be fine.